esp-link/cgimqtt.c

```diff
@@ -17,17 +17,29 @@ int ICACHE_FLASH_ATTR cgiMqttGet(HttpdConnData *connData) {
       "\"slip-enable\":%d, "
       "\"mqtt-enable\":%d, "
       "\"mqtt-status-enable\":%d, "
+      "\"mqtt-clean-session\":%d, "
```

Comment:
I don't think it makes sense to support clean_session at the moment. Would need a bunch of changes to the mqtt code.